src/projetos/SugestaoPage.tsx

```
@@ -287,14 +303,19 @@ export function SugestaoPage() {
-                        </div>
-                        <div className="flex justify-end">
-                            <Button
-                                type="submit"
-                                className="!bg-yellow-600 hover:!bg-yellow-700 text-black"
-                            >
-                                {id ? 'Atualizar Projeto' : 'Cadastrar Projeto'}
-                            </Button>
+
+                            <div className="space-y-6">
+                                {/* Regras*/}
+                                <div className='mt-6 bg-gray-100 w-full h-[150px] border rounded-xl'> escrever alguma coisa</div>
```

ESLint

[error] Replace `································<div·className='mt-6·bg-gray-100·w-full·h-[150px]·border·rounded-xl'>·escrever·alguma·coisa</div>` with `················<div·className="mt-6·bg-gray-100·w-full·h-[150px]·border·rounded-xl">⏎··················{'·'}⏎··················escrever·alguma·coisa`

(prettier/prettier)